src/components/step-nav.scss

```diff
@@ -0,0 +1,197 @@
+@import "../base/config.scss";
+
+//------------------------------------------------------------//
+
+// * StepNav Component
+// * 1. Settings
+// * 2. Define Default StepNav
+// * 3. Small Modifier
+// * 4. Title Modifier
+// * 5. Responsive Mixin
+
+// TODO : document global dependencies
+// TODO : Add Small StepNav Modifier
+// TODO : Add Titled Modifier
+
+//------------------------------------------------------------//
+
+// * 1. Settings
+
+$StepNav-size: 60px;
+$StepNav-border: 4px;
+
+$StepNav-size--small: 40px;
+$StepNav-border--small: 2px;
+
+// * 2. Define Default StepNav
+
+.StepNav {
+  padding-left: ($StepNav-size * 1.25);
+  position: relative;
+  @include clear();
+  padding-bottom: $vertical-block-spacing--small;
+  &.steps-2 {
+    .StepNav-Step {
+      width: 100%;
+    }
+  }
+  &.steps-3 {
+    .StepNav-Step {
+      width: 50%;
+    }
+  }
+  &.steps-4 {
+    .StepNav-Step {
+      width: 33%;
+    }
+  }
+}
+
+.StepNav-Step {
```

Comment:
[Children](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#ComponentName-descendentName) and modifiers use camel case, so it would be `.StepNav-step`, `.StepNav-item`.